package.json

```diff
@@ -40,8 +39,12 @@
   "devDependencies": {
     "@commitlint/cli": "17.0.3",
     "@commitlint/config-conventional": "17.0.3",
+    "ds-test": "github:dapphub/ds-test#e282159",
+    "forge-std": "github:foundry-rs/forge-std#v1.5.6",
     "husky": ">=8",
+    "isolmate": "github:defi-wonderland/isolmate#59e1804",
     "lint-staged": ">=10",
+    "prb/test": "github:paulrberg/prb-test#a245c71",
```

Comment:
`isolmate` and `prb-test` are production dependencies. I would add `ds-test` and `forge-std` as production too.

Reply:
are they production tho? (is production the deployed contracts?)
if so the production contract uses those on compilation, which is dev.

Reply:
@skeletor-spaceman my comment mostly inspired by the experience of importing price interfaces package in keep3r scripts and other repos. E.g. having `isolmate` as a dev dependency means the importing party will have to explicitly list it as their dependency or see `yarn install` failures.

Reply:
oh then it is most definitely required. I agree. nice one!

Reply:
smart gas. Fixed 5bb9640d0b141d9fa3f2704afa6df28d12372e14